pkg/util/graph/adapter.go

```diff
@@ -0,0 +1,36 @@
+package graph
```

Comment:
I might be a little confused due to commit order, but would these files be required for the `make` target to work? If so it might be clearer to squash those two together as the first commit. I'm trying to figure out what is a root dependency, generated content, or other changes we need to use the generated SDK. This feels like a root dependency for generation since it's importing `kiota` stuff?

Reply:
Not particularly -- none of this works until it's all done :). These files wrap the generated code for us. The kiota generator outputs go but is not go, so it doesn't matter if the codebase is in an inconsistent state between these commits.